docs/src/installation.md

```diff
@@ -1,85 +1,124 @@
+| [CSDP](https://github.com/coin-or/Csdp)                                        | [CSDP.jl](https://github.com/jump-dev/CSDP.jl)                                   |     | EPL      | LP, SDP                         |
+| [EAGO.jl](https://github.com/psorlab/EAGO.jl)                                  |                                                                                  |     | CC BY-NC-SA | NLP                          |
+| [ECOS](https://github.com/ifa-ethz/ecos)                                       | [ECOS.jl](https://github.com/jump-dev/ECOS.jl)                                   |     | GPL      | LP, SOCP                        |
+| [FICO Xpress](https://www.fico.com/en/products/fico-xpress-optimization-suite) | [Xpress.jl](https://github.com/jump-dev/Xpress.jl)                               | Yes | Comm.    | (MI)LP, (MI)SOCP                |
+| [GLPK](http://www.gnu.org/software/glpk/)                                      | [GLPK.jl](https://github.com/jump-dev/GLPK.jl)                                   |     | GPL      | (MI)LP                          |
+| [Gurobi](https://gurobi.com)                                                   | [Gurobi.jl](https://github.com/jump-dev/Gurobi.jl)                               | Yes | Comm.    | (MI)LP, (MI)SOCP                |
+| [Hypatia.jl](https://github.com/chriscoey/Hypatia.jl)                          |                                                                                  |     | MIT      | LP, SOCP, SDP                   |
+| [Ipopt](https://github.com/coin-or/Ipopt)                                      | [Ipopt.jl](https://github.com/jump-dev/Ipopt.jl)                                 |     | EPL      | LP, QP, NLP                     |
+| [Juniper.jl](https://github.com/lanl-ansi/Juniper.jl)                          |                                                                                  |     | MIT      | (MI)SOCP, (MI)NLP               |
+| [MOSEK](https://www.mosek.com/)                                                | [MosekTools.jl](https://github.com/jump-dev/MosekTools.jl)                       | Yes | Comm.    | (MI)LP, (MI)SOCP, SDP           |
+| [NLopt](https://github.com/stevengj/nlopt)                                     | [NLopt.jl](https://github.com/JuliaOpt/NLopt.jl)                                 |     | GPL      | LP, QP, NLP                     |
+| [OSQP](https://osqp.org/)                                                      | [OSQP.jl](https://github.com/oxfordcontrol/OSQP.jl)                              |     | Apache   | LP, QP                          |
+| [Pavito.jl](https://github.com/jump-dev/Pavito.jl)                             |                                                                                  |     | MPL-2    | (MI)CP                          |
```

Comment:
CP is not defined. It should be (MI)NLP.
The additional info cam state that it works globally if nlp is convex.